ArduPlane/Parameters.h

```diff
@@ -345,8 +345,11 @@ class Parameters {
         k_param_pidWheelSteer, // unused
 
         k_param_mixing_offset,
-        k_param_dspoiler_rud_rate,
-
+        
+        // 250: ArduSoar parameters
+        k_param_soaring_controller,
```

Comment:
OK, I've reintroduced this change, please take a look. Note that the tests _may_ pass, but the issue is definitely still there. I don't have a very good grasp of how the G2 group of parameters works, but it is clear that it works differently than the standard one.